Paper.tex

```diff
@@ -425,77 +482,82 @@ \subsubsection{Block Header Validity}
 
-This is the foundation of the security of the blockchain and is the fundamental reason why a malicious node cannot propagate newly created blocks that would otherwise overwrite (``rewrite'') history. Because the nonce must satisfy this requirement, and because its satisfaction depends on the contents of the block and in turn its composed transactions, creating new, valid, blocks is difficult and, over time, requires approximately the total compute power of the trustworthy portion of the mining peers.
+This is the foundation of the security of the blockchain and is the fundamental reason why a malicious node cannot propagate newly created blocks that would otherwise overwrite (``rewrite'') history. Because the \hyperlink{block nonce}{block nonce} must satisfy this requirement, and because its satisfaction depends on the contents of the block and in turn its composed transactions, creating new, valid, blocks is difficult and, over time, requires approximately the total compute power of the trustworthy portion of the mining peers.
 
-Thus we are able to define the block header validity function $V(H)$:
+Thus we are able to define the \hypertarget{BlockHeaderValidityFunction}{block header validity function $V(H)$}:\label{eq:BlockHeaderValidityFunction}
 \begin{eqnarray}
-V(H) & \equiv &  n \leqslant \frac{2^{256}}{H_d} \wedge m = H_m \quad \wedge \\
-& & H_d = D(H) \quad \wedge \\
-& & H_g \le H_l  \quad \wedge \\
-& & H_l < {P(H)_H}_l + \left\lfloor\frac{{P(H)_H}_l}{1024}\right\rfloor  \quad \wedge \\
-& & H_l > {P(H)_H}_l - \left\lfloor\frac{{P(H)_H}_l}{1024}\right\rfloor  \quad \wedge \\
-& & H_l \geqslant 5000  \quad \wedge \\
-& & H_s > {P(H)_H}_s \quad \wedge \\
-& & H_i = {P(H)_H}_i +1 \quad \wedge \\
-& & \lVert H_x \rVert \le 32
+V(H) & \equiv &  n \leqslant \frac{2^{256}}{\hyperlink{H d}{H_{\mathrm{d}}}} \wedge m = \hyperlink{H m}{H_{\mathrm{m}}} \quad \wedge \\
+& & \hyperlink{H d}{H_{\mathrm{d}}} = D(H) \quad \wedge \\
+& & \hyperlink{H g}{H_{\mathrm{g}}} \le \hyperlink{H l}{H_{\mathrm{l}}}  \quad \wedge \\
+& & \hyperlink{H l}{H_{\mathrm{l}}} < {P(H)_H}_{\mathrm{l}} + \left\lfloor\frac{{P(H)_H}_{\mathrm{l}}}{1024}\right\rfloor  \quad \wedge \\
+& & \hyperlink{H l}{H_{\mathrm{l}}} > {P(H)_H}_{\mathrm{l}} - \left\lfloor\frac{{P(H)_H}_{\mathrm{l}}}{1024}\right\rfloor  \quad \wedge \\
+& & \hyperlink{H l}{H_{\mathrm{l}}} \geqslant 125000  \quad \wedge \\
+& & \hyperlink{H s}{H_{\mathrm{s}}} > {P(H)_H}_{\mathrm{s}} \quad \wedge \\
+& & \hyperlink{H i}{H_{\mathrm{i}}} = {P(H)_H}_{\mathrm{i}} +1 \quad \wedge \\
+& & \lVert \hyperlink{H x}{H_{\mathrm{x}}} \rVert \le 32
```

Comment:
These have all been referenced fairly recently so it seems unnecessary to link to them again.